datasets/bookcorpusopen/bookcorpusopen.py

```diff
@@ -0,0 +1,98 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors and the HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""The BookCorpus dataset based on Shawn Presser's work https://github.com/soskek/bookcorpus/issues/27 """
+
+from __future__ import absolute_import, division, print_function
+
+import glob
+import pathlib
+
+import datasets
+
+
+_DESCRIPTION = """\
+Books are a rich source of both fine-grained information, how a character, \
+an object or a scene looks like, as well as high-level semantics, what \
+someone is thinking, feeling and how these states evolve through a story.\
+This work aims to align books to their movie releases in order to provide\
+rich descriptive explanations for visual content that go semantically far\
+beyond the captions available in current datasets. \
```

Comment:
Could you add information like the number of books available ?
Maybe add credits to eleuther.ai in the description, and details from the origin of the data that they used ?